packages/core/useTextareaAutosize/demo.vue

```diff
@@ -0,0 +1,17 @@
+<script setup lang="ts">
+import { useTextarea } from '@vueuse/core'
+
+const { textarea, input } = useTextarea()
```

Comment:
```suggestion
const { textarea, input } = useTextareaAutoResize()
```

Reply:
Good catch, renamed it in a few other places I missed too